members/models.py

```diff
@@ -0,0 +1,56 @@
+from core.models import *
+
+
+# Create your models here.
+class Member(models.Model):
+    """
+    The member model.
+
+    It contains:
+        -an optional reference to the user if it already exists.
+        -the semester the membership was sold in
+            (and for ordinary members also the semester it's valid for)
+        -if the member got a lifetime or honorary membership.
+        -the date the membership was sold
+        -who sold it
+        -name of the member
+        -email address that can be blank
+
+    Can only be one member with the same name and email address per semester.
+    """
+    user = models.ForeignKey(User, null=True, blank=True, related_name='user')
```

Comment:
related_name er navnet User-objektet får til å peke tilbake på Member. Det gir ikke mening at man skal bruke User.user for å finne en Member. Kan nok fint sløyfes, i så fall blir det som å skrive 'member_set'. Se https://docs.djangoproject.com/es/1.9/topics/db/queries/#backwards-related-objects